packages/azure-container-services/src/activate.js

```diff
@@ -0,0 +1,60 @@
+/*
+ * (c) Copyright IBM Corp. 2023
+ */
+
+'use strict';
+
+const instanaCore = require('@instana/core');
+const { backendConnector, consoleLogger } = require('@instana/serverless');
+
+const identityProvider = require('./identity_provider');
+
+const { tracing, util: coreUtil } = instanaCore;
+const { normalizeConfig } = coreUtil;
+
+let logger = consoleLogger;
+
+const config = normalizeConfig({});
+
+function init() {
+  if (process.env.INSTANA_DEBUG || process.env.INSTANA_LOG_LEVEL) {
+    logger.setLevel(process.env.INSTANA_DEBUG ? 'debug' : process.env.INSTANA_LOG_LEVEL);
+  }
+  // For more details about environment variables in azure, please see
+  // https://learn.microsoft.com/en-us/azure/app-service/reference-app-settings?tabs=kudu%2Cdotnet#app-environment
+  if (!process.env.WEBSITE_OWNER_NAME && !process.env.WEBSITE_SITE_NAME && !process.env.WEBSITE_RESOURCE_GROUP) {
+    logger.error(
+      'Initializing @instana/azure-container-services failed. The environment variables' +
+        `${process.env.WEBSITE_OWNER_NAME} ${process.env.WEBSITE_SITE_NAME} ${process.env.WEBSITE_RESOURCE_GROUP} ` +
```

Comment:
Suggestion: The environment variables are WEBSITE_OWNER_NAME: ${process.env.WEBSITE_OWNER_NAME}, WEBSITE_SITE_NAME: ${process.env.WEBSITE_SITE_NAME} and WEBSITE_RESOURCE_GROUP: ${process.env.WEBSITE_RESOURCE_GROUP}.

Reply:
done